app/src/androidTest/java/com/github/se/project/ui/lesson/RequestedLessonsTest.kt

```diff
@@ -146,76 +183,33 @@ class LessonsRequestedScreenTest {
   @Test
   fun testDatePickerButtonFunctionality() {
     composeTestRule.setContent {
-      RequestedLessonsScreen(listProfilesViewModel, lessonViewModel, navigationActions)
+      RequestedLessonsScreen(
+          listProfilesViewModel, lessonViewModel, mockNetworkStatusViewModel, navigationActions)
     }
 
     // Click on the DatePicker to ensure it can be interacted with
     composeTestRule.onNodeWithTag("datePicker").performClick()
   }
 
-  @Test
-  fun testLessonsFilteredBySelectedDate() {
-    // Set the selected date to filter lessons
-    val filteredDate = "2024-10-10"
-
```

Comment:
Not sure to understand why this test was removed? Good you not make it work?

Reply:
This test had many flaws and would never work as such and what it was testing was already covered by the other tests, that's why I remove it